client/cli/cli.go

```diff
@@ -63,27 +63,52 @@ func Run(run bool, pid int, args []string) {
 	}()
 
 	cmds := command.DebugCommands()
+	names := cmds.Names()
+
+	// set command completer
+	t.line.SetWordCompleter(func(line string, pos int) (head string, completions []string, tail string) {
+		start := strings.LastIndex(line[:pos], " ")
```

Comment:
Is this checking to see if we're trying to complete a subcommand, i.e. `info args` or something? I think those are going away (the `info` part of the last command) so I think only the second conditional is relevant. Also, I think we could get rid of the early return as well.